packages/terra-layout/src/LayoutSlidePanel.module.scss

```diff
@@ -39,28 +39,49 @@
   }
 
   .panel.is-tiny {
-    width: var(--layout-slide-panel-phone-panel-width, calc(100% - 3.571rem));
+    width: var(--terra-layout-slide-panel-tiny-panel-width, calc(100% - 3.571rem));
   }
 
   .panel.is-small {
-    width: var(--layout-slide-panel-tablet-panel-width, 22.857rem);
+    width: var(--terra-layout-slide-panel-small-panel-width, 22.857rem);
   }
 
   // Overlay
   .is-open.is-overlay > .panel {
-    box-shadow: 0.07143rem 0 0.3571rem var(--layout-slide-panel-panel-shadow-color, #1c1f21);
+    /* stylelint-disable selector-max-compound-selectors */
+    [dir=ltr] & {
+      box-shadow: var(--terra-layout-slide-panel-overlay-panel-ltr-box-shadow, 1px 0 0 0 #1c1f21, 0 0 5px 1px #000, 0 0 12px 2px rgba(0, 0, 0, 0.8));
+    }
+
+    [dir=rtl] & {
+      box-shadow: var(--terra-layout-slide-panel-overlay-panel-rtl-box-shadow, -1px 0 0 0 #1c1f21, 0 0 5px 1px #000, 0 0 12px 2px rgba(0, 0, 0, 0.8));
+    }
+    /* stylelint-enable selector-max-compound-selectors */
   }
 
   // Squish
   .is-open.is-squish {
     > .content {
-      width: calc(100% - #{var(--layout-slide-panel-desktop-panel-width, 18.571rem)} - 0.07143rem);
+      width: calc(100% - #{var(--terra-layout-slide-panel-panel-width, 18.571rem)} - #{var(--terra-layout-slide-panel-squish-panel-border-right, 1px)});
     }
 
     > .panel {
-      border-right: 0.07143rem solid var(--layout-slide-panel-panel-border-color, #d3d4d5);
+      background-color: var(--terra-layout-slide-panel-squish-panel-background-color, #fff);
+      border-right-color: var(--terra-layout-slide-panel-squish-panel-border-right-color, #d3d4d5);
+      border-right-style: solid;
+      border-right-width: var(--terra-layout-slide-panel-squish-panel-border-right-width, 1px);
       transition: none;
-      width: calc(0.07143rem + #{var(--layout-slide-panel-desktop-panel-width, 18.571rem)});
+      width: calc(#{var(--terra-layout-slide-panel-squish-panel-border-right, 1px)} + #{var(--terra-layout-slide-panel-panel-width, 18.571rem)});
```

Comment:
I think this should be updated to `--terra-layout-slide-panel-squish-panel-border-right-width`.  If we leave the width off, people may think they can set this value to `2px solid #f00` where we just want the width value to use in the calculation.